app/routes/goal.py

```diff
@@ -0,0 +1,146 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.goal import Goal
+from app import db 
+from datetime import datetime
+from app.routes.task import validate_task 
+
+
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+# helper function to validate_goal
+def validate_goal(goal_id):
+    try:
+        goal_id = int(goal_id)
+    except:
+        abort(make_response({"message":f"Goal {goal_id} invalid"}, 400))
+
+    goal = Goal.query.get(goal_id)
+
+    if not goal:
+        abort(make_response({"message":f"Goal {goal_id} not found"}, 404))
+
+    return goal
+
+
+# Routes/Endpoints below 
+
+# create one goal
+@goals_bp.route("", methods=["POST"])
+def create_one_goal():
+    request_body = request.get_json()
+
+    if "title" not in request_body:
+        abort(make_response({"details": "Invalid data"}, 400))
+    else:
+        new_goal = Goal(title=request_body["title"])
+
+    db.session.add(new_goal)
+    db.session.commit()
```

Comment:
Great job using `db.session.<method-name>.` Session gives us access to the following:

* `db` is our app’s instance of SQLAlchemy.
* `session` represents our active database connection.
* By referencing db.session we can use SQLAlchemy’s methods to perform tasks like committing a change to a model, storing a new record of a model, and deleting a record.

By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the Goal model. ⭐